client/components/App.jsx

```diff
@@ -39,6 +40,26 @@ class App extends Component {
     zoomToken: '',
   };
 
+  static getInterviewDetails = (description) => {
+    const FIRST_IDX = 5;
+    const LAST_IDX = 6;
+    const EMAIL_IDX = 7;
+    const CHAT_IDX = 3;
+
+    // Determine whether the interview is based on the newer format
+    const isNewer = description.indexOf(interviewsNewMetadata) >= 0;
```

Comment:
`description.includes(interviewsNewMetadata)` would be more descriptive.

Reply:
Now using includes instead of indexOf